mantidproject/mantid
# Instrument view extensions

**Description of work.**
This PR adds the following : 
- possibility to mask a single pixel of a detector or a whole tube / bank / component
- removes the integration bar on the instrument render tab for monochromatic workspaces, as it is useless.
- reduces the plot sum spectra panel on the instrument pick tab when working on monochromatic workspaces.
- adds a reset view button to restore the default projection

WIP for #28352 


**To test:**


Try these features on a monochromatic workspace (eg ILL/D16/218356.nxs) and a regular one, and check that everything works as intended.
Also check Mantidplot, as the integration bar is managed differently there.





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
